.ci/jobs.yml

```diff
@@ -1,7 +1,24 @@
 JOB:
-  - selenium
-  - intake
-  - x-pack
+  - kibana-intake
```

Comment:
Can we also add a note to also define in `test/scripts/jenkins_xpack_ci_group.sh` and `tasks/function_test_groups.js`